schemas/top-api.yml

```diff
@@ -1671,6 +1669,12 @@ paths:
           schema:
             type: string
           example: snomed
+        - name: subtree
```

Comment:
Why introduce another parameter name here? I would stick to 'scope' or 'codeScope'.

Reply:
Resolved by 898496346ceb7577233d44771010fced11a04a6b